user_interaction/views.py

```diff
@@ -95,7 +95,14 @@ def get_context_data(self, **kwargs):
             welcome_name = optimise_naming_scheme(welcome_name)
 
         activities = []
-        for activity in Activity.objects.filter(published_date__lte=timezone.now(), type=ActivityType.ACTIVITY_PUBLIC):
+        for activity in (
+            Activity.objects.filter(published_date__lte=timezone.now())
+            .filter(
+                Q(type=ActivityType.ACTIVITY_PUBLIC)
+                | (Q(type=ActivityType.ACTIVITY_MEETING) & Q(organisers__members=self.request.member))
```

Comment:
I'm not sure what will happen if `self.request.member = None`. Might be useful to check that condition first just to be safe. Something along the lines of (pseudocode):
```python
activities = Activity.objects.filter(PUBLIC-Q)
if member is not None:
    activities = activities.filter(MEETINGS-Q)
for activity in activities:
    ...
```